config/params_template.env

```diff
@@ -44,12 +44,19 @@ export src_subdiv_toggle="True" # Toggle to run composite roughness src routine
 export vrough_suffix="" # text to append to output log and hydrotable file names (use for testing/debugging)
 export vmann_input_file="/data/inputs/rating_curve/variable_roughness/mannings_global_06_12.csv" # input file location with nwm feature_id and channel roughness and overbank roughness attributes
 
-#### apply SRC adjustments using USGS rating curve database (calibration db) ####
-export src_adjust_usgs="True" # Toggle to run src adjustment routine (True=on; False=off)
```

Comment:
`src_adjust_usgs` is still referenced on line 131 in `fim_post_processing.sh`. If we delete it here, then the call to `src_adjust_usgs_rating.py` on line 136 will not take place. Not sure if this is intentional or not.

Reply:
`src_adjust_usgs` is defined below (line 51).  I did some reorganization of these calibration parameters, but I don't think I deleted anything.

Reply:
Good deal, when previously viewing this PR, my screen didn't show line 51. Thanks!